pkg/actuators/machineset/controller.go

```diff
@@ -111,7 +112,10 @@ func reconcile(machineSet *machinev1.MachineSet) (ctrl.Result, error) {
 	}
 	instanceType, ok := InstanceTypes[providerConfig.InstanceType]
 	if !ok {
-		return ctrl.Result{}, mapierrors.InvalidMachineConfiguration("unknown instance type: %s", providerConfig.InstanceType)
+		klog.Error("Unable to set scale from zero annotations: unknown instance type: %s", providerConfig.InstanceType)
+		klog.Error("Autoscaling from zero will not work. To fix this, manually populate machine annotations for you instance type\n- %s\n- %s\n -%s",
+			cpuKey, memoryKey, gpuKey)
```

Comment:
Having new lines in the log line could have unintended consequences, best to avoid IMO, you could print as a list instead, this should give something like `[blah, blah, blah]` in the line instead
```suggestion
		klog.Error("Autoscaling from zero will not work. To fix this, manually populate machine annotations for you instance type: %v", []string{cpuKey, memoryKey, gpuKey})
```